### DISCUSSION TITLE
Fix Mo' Creatures' flickering FlameWraith

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#165] [Bugfix Suggestions]
https://user-images.githubusercontent.com/8945347/235361874-8c46db6b-a789-4dee-bfa8-a3634f9580f3.mp4

Distractingly buggy.

**Comment by ACGaming** [comment]
Implemented in 3789f94117ed0fd91fd1e575ae12b7ce4d75ab83.
